e2e/test_module.py

```diff
@@ -13,15 +14,18 @@ def db():
 class TestConstants:
     ABSOLUTE_TOLERANCE = 1e-3
 
+    CLEANUP = "cleanup"
     EXCEPTION = "exception"
     INPUT_FILE = "input.cyp"
     OUTPUT = "output"
+    ONLINE_TEST = "_test"
+    SETUP = "setup"
     QUERY = "query"
+    QUERIES = "queries"
```

Comment:
Maybe different constant name here. Because it was hard for me to debug what are we using it for.

For example:
ONLINE_TEST_QUERIES_KEYWORD

Reply:
or ONLINE_TEST_E2E_INPUT_QUERIES